pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -2,238 +2,198 @@
+PWA Studio storefronts require you to set the following [environment variables][] in your Cloud project:
 
-```yaml
-dependencies:
-    nodejs:
-      yarn: "*"
-```
+| Name                                 | Description                                          |
+| ------------------------------------ | ---------------------------------------------------- |
+| `CONFIG__DEFAULT__WEB__UPWARD__PATH` | Absolute path to UPWARD YAML configuration           |
+| `NODE_ENV`                           | Specifies the node environment type                  |
+| `MAGENTO_BACKEND_URL`                | URL of your Magento backend                          |
+| `CHECKOUT_BRAINTREE_TOKEN`           | Braintree token associated with your Magento backend |
+| `MAGENTO_BACKEND_EDITION`            | Either `CE` or `EE`                                  |
+| `IMAGE_OPTIMIZING_ORIGIN`            | Origin to use for images in the UI                   |
 
-### Update build hooks
+Magento Cloud offers a variety of ways to set environment variables, but the most direct way is to edit the [`.magento.app.yaml`][] file and add entries to the `variables.env` section.
 
-The `venia-concept` storefront uses a newer version of Node than the default environment provides.
-It also uses Yarn to install its dependencies.
+```text
+variables:
+    env:
+        CONFIG__DEFAULT__WEB__UPWARD__PATH: "/app/pmu35riuj7btw_stg/pwa/upward.yml"
+        NODE_ENV: "production"
+        MAGENTO_BACKEND_URL: "https://[your-cloud-url-here]/"
+        CHECKOUT_BRAINTREE_TOKEN: "<generated token from Braintree>"
+        MAGENTO_BACKEND_EDITION: "EE"
+        IMAGE_OPTIMIZING_ORIGIN: "backend"
+```
 
-The following update to the `hooks.build` section of `.magento.app.yaml` installs NVM and the latest LTS release of Node.
+### Finding the correct UPWARD path value
 
-It also uses NPX to run the `buildpack` CLI tool with the `create-project` sub-command to scaffold the application using the `venia-concept` template.
-After scaffolding completes, it installs and builds the application and moves the build artifact to a persistent area on the file system.
+The `CONFIG_DEFAULT_WEB_UPWARD_PATH` variable specifies the _absolute_ path to the UPWARD configuration file in the deployed instance.
+If this value is incorrect or not set, the Magento 2 UPWARD connector extension cannot serve your storefront application and your frontend appears broken.
 
-```yaml
-hooks:
-    # We run build hooks before your application has been packaged.
-    build: |
-        set -e
+In the previous example, `/app/pmu35riuj7btw_stg/` is the Magento application root directory on the deployed instance.
+This value is different for each environment in your Cloud project, so you must configure each of your project environments with the path specific to each instance.
+To find the correct root directory path for an environment, [SSH][] into the remote server and use the `pwd` command to find the Magento application root directory.
```

Comment:
Does the `pwd` command show the directory? Or do they need to traverse at all?

Reply:
good question. @dpatil-magento when you SSH into a cloud server, does it just dump you into the project root directory or do you have to navigate to it?

Reply:
You're dumped into your home `~` when you login, which is the application root. This step appears accurate to me.